src/gsy_e/gsy_e_core/util.py

```diff
@@ -443,21 +475,39 @@ def get_market_maker_rate_from_config(next_market, default_value=None, time_slot
                 raise e
         return find_object_of_same_weekday_and_time(GlobalConfig.market_maker_rate,
                                                     time_slot)
-    else:
-        return GlobalConfig.market_maker_rate
+    return GlobalConfig.market_maker_rate
+
+
+def get_feed_in_tariff_rate_from_config(next_market, default_value=None, time_slot=None):
+    """Get feed in tariff rate from config."""
+    if next_market is None:
+        return default_value
+    if isinstance(GlobalConfig.FEED_IN_TARIFF, dict):
```

Comment:
When will the `GlobalConfig.FEED_IN_TARIFF` be a dict? I do not see any mechanism that converts from float to dict. Can you maybe help me?

Reply:
I see your comment now: https://github.com/gridsingularity/gsy-e/pull/1338/files#r767532113 Thanks!

Reply:
https://github.com/gridsingularity/gsy-e/pull/1338/files#r767532113